api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/repository/v1/SdcDuplicateRepository.java

```diff
@@ -39,6 +39,16 @@ public interface SdcDuplicateRepository extends JpaRepository<SdcDuplicateEntity
         """)
     List<SdcDuplicateEntity> findAllUnresolvedNonAllowableEnrollmentDuplicatesForCurrentCollection();
 
+    @Query("""
+        SELECT sde FROM SdcDuplicateEntity sde
+        WHERE sde.collectionID = (SELECT C.collectionID FROM CollectionEntity C WHERE C.collectionStatusCode != 'COMPLETED')
```

Comment:
I don't think the collectionID check is required since the studentID is unique to the collection.